index.html

```diff
@@ -346,6 +345,47 @@ <h2>Skills</h2>
 
         </div>
     </div>
+  
+  
+  
+    <footer>
+        <div>
+            <p id="footer-text">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <form action="./index.html">
+            <label>
+                <input type="text" name="user" placeholder="Full name">
+            </label>
+            <label>
+                <input type="email" name="email" placeholder="Email Address">
+            </label>
+            <label>
+                <textarea>Email text here</textarea>
```

Comment:
- [ ] Kindly make sure has the `maxlength="500"` attribute as specified in the requirements also correct the message displayed in the text area, it is not emailed text here, but message text here.